app/assets/stylesheets/_turbolinks.scss

```diff
@@ -0,0 +1,5 @@
+@import "variables";
+
+.turbolinks-progress-bar {
```

Comment:
_The progress bar is enabled by default. It appears automatically for any page that takes longer than 500ms to load._

Si on bosse bien elle devrait disparaitre toute seule 😃.